test/rules.js

```diff
@@ -48,13 +52,13 @@ const equivalentArray = function (a1, a2) {
  * Assert that metadata detected in a spec is equal to the expected values.
  *
  * @param {String} url - public URL of a spec.
- * @param {String} file - name of local file containing a spec (without path and without ".html" suffix).
+ * @param {String} file - name of local file containing a spec (without pth and without ".html" suffix).
```

Comment:
wrong global replace
```suggestion
 * @param {String} file - name of local file containing a spec (without path and without ".html" suffix).
```